README.md

```diff
@@ -18,210 +18,181 @@ Darkflame Universe is licensed under AGPLv3, please read [LICENSE](LICENSE). Som
+You can do so with the following commands.
 
-:: Run CMake to generate make files
-cmake ..
+[Source of the below commands](https://askubuntu.com/questions/355565/how-do-i-install-the-latest-version-of-cmake-from-the-command-line)
 
-:: Run CMake with build flag to build
-cmake --build . --config Release
-```
-#### Windows for ARM has not been tested but should build by doing the following
-```batch
-:: Create the build directory
-mkdir build
-cd build
+```bash
+# Remove the old version of CMake
+sudo apt purge --auto-remove cmake
 
-:: Run CMake to generate make files
-cmake .. -DMARIADB_BUILD_SOURCE=ON
+# Prepare for installation
+sudo apt update && sudo apt install -y software-properties-common lsb-release && sudo apt clean all
 
-:: Run CMake with build flag to build
-cmake --build . --config Release
-```
+# Obtain a copy of the signing key
+wget -O - https://apt.kitware.com/keys/kitware-archive-latest.asc 2>/dev/null | gpg --dearmor - | sudo tee /etc/apt/trusted.gpg.d/kitware.gpg >/dev/null
 
-### Windows builds (WSL)
-This section will go through how to install [WSL](https://docs.microsoft.com/en-us/windows/wsl/install) and building in a Linux environment under Windows. WSL requires Windows 10 version 2004 and higher (Build 19041 and higher) or Windows 11.
+# Add the repository to your sources list.
+sudo apt-add-repository "deb https://apt.kitware.com/ubuntu/ $(lsb_release -cs) main"
 
-#### Open the Command Prompt application with Administrator permissions and run the following:
-```bash
-# Installing Windows Subsystem for Linux
-wsl --install
-```
+# Next you'll want to ensure that Kitware's keyring stays up to date
+sudo apt update
+sudo apt install kitware-archive-keyring
+sudo rm /etc/apt/trusted.gpg.d/kitware.gpg
 
-#### Open the Ubuntu application and run the following:
-```bash
-# Make sure the install is up to date
-apt update && apt upgrade
+# If sudo apt update above returned an error, copy the public key at the end of the error message and run the following command
+# if the error message was "The following signatures couldn't be verified because the public key is not available: NO_PUBKEY 6AF7F09730B3F0A4"
+# then the below command would be "sudo apt-key adv --keyserver keyserver.ubuntu.com --recv-keys 6AF7F09730B3F0A4"
+sudo apt-key adv --keyserver keyserver.ubuntu.com --recv-keys <TheCopiedPublicKey>
 
-# Make sure the gcc, cmake, and build-essentials are installed
-sudo apt install gcc
+# Finally update and install
+sudo apt update
 sudo apt install cmake
-sudo apt install build-essential
 ```
 
-[**Follow the Linux instructions**](#linux-builds)
+## Database setup
+First you'll need to start MariaDB.
 
-### ARM builds
-AArch64 builds should work on linux and MacOS using their respective build steps. Windows ARM should build but it has not been tested
+For Windows the service is always running by default.
```

Comment:
Maybe make it clear somewhere in the Database setup that the MariaDB install needs the be on the default port `3306`. 

On Windows it allows the user to change the port on install, or will change it automatically if `3306` is in use. I've come across multiple users who either changed the default port on accident or on purpose and led to issues.

Reply:
I think it would be easier to make it a part of the database socket instead so people can change it if they want to